pyiron_base/database/generic.py

```diff
@@ -222,6 +223,53 @@ def job_table(
     def _get_table_headings(self, table_name=None):
         pass
 
+    def item_update(self, par_dict, item_id):
+        return self._item_update(par_dict=par_dict, item_id=item_id)
+
+    @abstractmethod
+    def _item_update(self, par_dict, item_id):
+        pass
+
+    def items_update(self, par_dict, item_ids):
+        return self._items_update(par_dict=par_dict, item_ids=item_ids)
+
+    def _items_update(self, par_dict, item_ids):
+        """
+        For now simply loops over all item_ids to call item_update,
+        but can be made more efficient.
+        Should be made an asbtract method when defined in inheriting classes
+
+        Args:
+            par_dict (_type_): _description_
+            item_ids (_type_): _description_
+        """
+        for i_id in item_ids:
+            self.item_update(par_dict=par_dict, item_id=i_id)
+
+    def set_job_status(self, status, job_id):
+        """
+        Set status of a job.
+
+        Args:
+            status (str): status
+            job_id (int): job id
+        """
+        self.item_update(
+            par_dict={"status": status},
+            item_id=job_id,
+        )
+
+    def set_jobs_status(self, status, job_ids):
+        """
+        Set status of multiple jobs
+
+        Args:
+            status (_type_): _description_
+            job_ids (_type_): _description_
+        """
+        for j_id in job_ids:
+            self.set_job_status(status=status, job_id=j_id)
+
```

Comment:
Can this be handled by `item_update()`?

Reply:
What exactly do you mean? set_job_status was just a convenient wrapper around item_update() in the case of DatabaseAccess anyway and in the case of FileTable the hdf5 writing part is the problem.

As we discussed above it would be nice to have an efficient implementation for set_jobs_status, but as long as that does not exist the for loop serves its purpose